src/VisualStudio/CSharp/Impl/xlf/CSharpVSResources.de.xlf

```diff
@@ -4,47 +4,47 @@
     <body>
       <trans-unit id="Add_accessibility_modifiers">
         <source>Add accessibility modifiers</source>
-        <target state="new">Add accessibility modifiers</target>
+        <target state="translated">Zugriffsmodifizierer hinzufügen</target>
         <note />
       </trans-unit>
       <trans-unit id="Add_remove_braces_for_single_line_control_statements">
         <source>Add/remove braces for single-line control statements</source>
-        <target state="new">Add/remove braces for single-line control statements</target>
+        <target state="translated">Geschweifte Klammern für einzeilige Steuerungsanweisungen hinzufügen/entfernen</target>
         <note />
       </trans-unit>
       <trans-unit id="Apply_all_csharp_formatting_rules_indentation_wrapping_spacing">
         <source>Apply all C# formatting rules (indentation, wrapping, spacing)</source>
-        <target state="new">Apply all C# formatting rules (indentation, wrapping, spacing)</target>
+        <target state="translated">Alle C#-Formatierungsregeln anwenden (Einzug, Umbruch, Abstand)</target>
```

Comment:
"Einzug" is used in typography and printing, "Einrückung" is used in programming. (See also: line [72 in `BasicVSResources.de.xlf`](https://github.com/dotnet/roslyn/blob/6c7c773ae6ea0ac85951e4c83c33db2ef6b3dd16/src/VisualStudio/VisualBasic/Impl/xlf/BasicVSResources.de.xlf#L72), the [German Wikipedia article on "Indentation style"](https://de.wikipedia.org/wiki/Einr%C3%BCckungsstil).)

Reply:
Thank you for your feedback. We will update transaltion during next round of localization